### PR TITLE
Add Ares-6 

### PR DESCRIPTION
**Pull request by meg-gupta**
Adding all the files required to run the cli version of Ares-6. 
The benchmark can be run from cli.js

### CODE REVIEW

**Comment by Cellule** [comment]
You must also update https://github.com/Microsoft/ChakraCore/blob/master/THIRD-PARTY-NOTICES.txt to include the license notice of the benchmark

**Comment by meg-gupta** [comment]
@Cellule thanks, updated

**Comment by Cellule** [comment]
> 

Maybe we should print `SCORE: <time> ms` to be more consistent with what some other benchmarks do

---
Refers to: test/benchmarks/ARES-6/Basic/test.js:17 in 55f37c8. [](commit_id = 55f37c88b0dee733f46878c78216913fa442532a, deletion_comment = False)

File: test/benchmarks/ARES-6/results.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    {
+        if (times.length < 5)
+            throw new Error("We expect >= 5 iterations");
+
+        this.firstIteration.add(times[0]);
+        let steadyTimes = times.slice(1).sort((a, b) => b - a); // Sort in reverse order.
+        this.averageWorstCase.add((steadyTimes[0] + steadyTimes[1] + steadyTimes[2] + steadyTimes[3]) / 4);
+        this.steadyState.add(steadyTimes.reduce((previous, current) => previous + current) / steadyTimes.length);
+        this.reportDone();
+    }
```
Comment (meg-gupta) [inline test/benchmarks/ARES-6/results.js]: @Cellule This prints the benchmark's results, from https://browserbench.org/ARES-6/about.html their results is not a single score, but they use 1. time for first iteration, 2. avg time for first 4 iterations and 3. avg time for all but first iteration.

**Pull request merged**